scss/style.scss

```diff
@@ -0,0 +1,1241 @@
+body {
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+}
```

Comment:
This should be part of `component-layout.scss` as we definitely wouldn't want to include this in the WP dashboard.

Reply:
@bradt Yeah good point. I think WP will use this anyway as it's default for system ui. Wil move to components-layout.scss

Reply:
Yes, but if WP updates its style our sheet could override it.